cfpb/design-stories
# [Interaction Tests] ExpandableGroup

Closes #36 

## Changes

- Expands on James' interaction testing to get them pass consistently on page load

## How to test this PR

1. Open preview
2. Go to [ExpandableGroup - default](https://deploy-preview-101--cfpb-design-stories.netlify.app/?path=/story/components-expandable-expandablegroup--default)
4. Reload page to verify interaction test run successfully on load
